atlp-rwanda/destructors-ec-fe
# #185106856 admin should be able to set roles/permissions

What does this PR do?

   - Admin should be able to set roles/permissions

Description of Task to be completed?

     As  superAdmin ,  you should be able to assign role  of  buyer, seller and admin  to a user  
    
How should this be manually tested?

  1. Clone the repository by running
     

   git clone https://github.com/atlp-rwanda/destructors-ec-fe.git


  2. Navigate to the project directory by running
  
   cd destructors-ec-fe

   git checkout ft-set-roles-185106856

  3. Install the necessary dependencies by running
  
       npm install

  4.  Run react development server by running
 
     npm run dev

  5. Run test by using this command
  
     npm run test

  6. Navigate to the app url htttp://localhost:/users
 
Any background context you want to provide?

          -   you should login as  superAdmin
          -    Backend server must be running
          
What are the relevant pivotal tracker/Trello stories?

Tracker: 185106856
